cfgov/cfgov/settings/base.py

```diff
@@ -35,6 +35,7 @@
 # Application definition
 
 INSTALLED_APPS = (
+    'permissions_viewer',
```

Comment:
Is there any specific reason why this is up here instead of down with our other apps?

Reply:
I think I was troubleshooting something when I moved it up there  (the real problem was me forgetting that the top level in urls.py needs to be called 'urlpatterns'). I'll fix it.